.github/workflows/test.yml

```diff
@@ -221,7 +221,7 @@ jobs:
 
       - uses: ./.github/actions/setup-turborepo-environment
         with:
-          windows: ${{ matrix.os.name == 'windows' }}
+          windows: ${{ matrix.os.runner == 'windows-latest' }}
```

Comment:
Was there an issue with the previous comparison? Just making sure I'm not missing something with this change.

Reply:
you aren't missing anything. I just realized that `runner` is a real thing in the matrix, where the `name` key is made up by the user (us), and could be dropped with no effect on the matrix. So I'd rather compare to the real thing. E.g in integration tests, I didn't even put a name , because it doesn't matter: https://github.com/vercel/turbo/blob/29e73add257b489decb33837003d545ad391f018/.github/workflows/test.yml#L255-L259